configs/katello_36.yaml

```diff
@@ -0,0 +1,181 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+  :3.5.0:
+    :redmine_release_id: 250
+  :3.5.1:
+    :redmine_release_id: 329
+:releases:
+  :3.6.0:
+    :redmine_release_id: 284
+:code_name: Imperial IPA
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-3.6
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :katello-selinux:
+    :branch: 3.0.3
+    :repo: https://github.com/Katello/katello-selinux.git
+  :katello-installer:
+    :branch: KATELLO-3.6
+    :repo: https://github.com/Katello/katello-installer.git
+    :version_branch: true
+  :hammer-cli-katello:
+    :branch: 0.11.4
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :foreman-packaging:
+    :branch: rpm/1.17
+    :repo: https://github.com/theforeman/foreman-packaging.git
+    :version_branch: true
+  :katello-agent:
+    :branch: 3.1.0
+    :repo: https://github.com/Katello/katello-agent.git
+  :bastion:
+    :branch: 6.1.0
+    :repo: https://github.com/Katello/bastion
+  :puppet-pulp:
+    :branch: 5.1.0
+    :repo: https://github.com/Katello/puppet-pulp
+  :puppet-foreman_proxy_content:
+    :branch: 6.0.0
+    :repo: https://github.com/Katello/puppet-foreman_proxy_content
+  :puppet-katello:
+    :branch: 7.0.0
+    :repo: https://github.com/Katello/puppet-katello
+  :puppet-candlepin:
+    :branch: 4.0.0
+    :repo: https://github.com/Katello/puppet-candlepin
+  :puppet-certs:
+    :branch: 3.0.1
+    :repo: https://github.com/Katello/puppet-certs
+
+:gpg_key: ''
+:tags:
+  - name: katello-thirdparty-candlepin-rhel7
+    based_off: null
+    arches:
+      - x86_64
+  - name: katello-thirdparty-pulp-rhel7
+    based_off: null
+    arches:
+      - x86_64
+  - name: katello-thirdparty-pulp-rhel6
+    based_off: null
+    arches:
+      - x86_64
+  - name: katello-thirdparty-pulp-rhel5
+    based_off: null
+    arches:
+      - x86_64
+  - name: katello-3.6-rhel7
+    based_off: null
+    helper_tags:
+      katello-thirdparty-rhel7: null
+      katello-3.6-rhel7-override: null
+      katello-client-rhel7: null
+    build_target: katello-3.6-rhel7-build
+    build_package_group_source_tag: katello-3.6-rhel7-build
+    arches:
+      - x86_64
+    inherits:
+      katello-3.6-rhel7-build:
+        katello-3.6-rhel7-override: 0
+        foreman-plugins-3.6-rhel7-override: 3
+        foreman-3.6-nonscl-rhel7: 4
+        foreman-3.6-rhel7: 10
```

Comment:
The Foreman version is 1.17, not 3.6. Check all tags in here.

Reply:
Derp, I %s/nightly/3.6, I'll change.